docs/interfaces/mxv/leds.md

```diff
@@ -0,0 +1,32 @@
+---
+title: LEDs (MX-V)
+tags:
+  - LED
+  - MX-V
+---
+
+## Overview
+
+This platform has red and greed LEDs.
+
+All leds can be used as you like and they are just named.
+
+**Warning:** GNSS led can be lit when powered and if it not turn off in 1 sec the main cpu has been hanged, the co-cpu will restart the system and if it continues the unit has failed to boot u-boot.
```

Comment:
If not turn off? Do you mean that if does not turn off in 1 sec there is an error?

Reply:
check again, i have rewritten the line